orion/core/dags/orion.py

```diff
@@ -1,3 +1,4 @@
+import torch
```

Comment:
Solved the dependency issues by importing torch at the beginning of the script. Various solutions discussed [here](https://github.com/pytorch/pytorch/issues/2575).